src/helpers.ts

```diff
@@ -17,8 +17,6 @@ import {
 import type { Encryption, ObjectMetaData, RequestHeaders } from './internal/type.ts'
 import { RETENTION_MODES } from './internal/type.ts'
 
-export { ENCRYPTION_TYPES, LEGAL_HOLD_STATUS, RETENTION_MODES, RETENTION_VALIDITY_UNITS } from './internal/type.ts'
```

Comment:
do not remove exports from `src/helpers.ts`, this may cause breaking change.

Reply:
Fixed in 40be1c9